api/client/ffs.go

```diff
@@ -325,6 +325,16 @@ func (f *ffs) WatchLogs(ctx context.Context, ch chan<- LogEvent, c cid.Cid, opts
 	return nil
 }
 
+func (f *ffs) SendFil(ctx context.Context, from string, to string, amount int64) error {
```

Comment:
Add some comment to describe that this send is async.
I'm starting to think we should comment all client APIs, because when the client tries to use a method it doesn't have much clue of what that _client_ API will do. 

I wonder why this public method doesn't bubble up in the golangci check.

Reply:
It's because the `ffs` struct isn't public (although it is exported through a public field on `Client`). I started doing this to keep people from using the `ffs` struct directly, forcing them instead to use `Client` to access `ffs` apis. Maybe it's not worth the way it breaks linting. I can PR the change that makes it public and adds missing comments. Let me know what you think

Reply:
Oh, I see... `ffs.Ffs` is a public attribute but the type is private.
Agree that might be better to make them public, since I think the comment methods for the client are a must for good DX.